### DISCUSSION TITLE
Integrate machine learning (reinforcement learning)

### DISCUSSION THREAD

**Discussion by Rom-1T** [riebl/artery#345] [Ideas]
Hello everyone,

I’m currently exploring the implementation of machine learning techniques for generating V2X messages, specifically Cooperative Perception Messages (CPM). I’m particularly interested in incorporating reinforcement learning.

Has anyone here experimented with integrating machine learning in Artery ? I came across a project that uses a wrapper for Omnet++ (available [here](https://github.com/ComNetsHH/omnetpp-ml)) and would love to hear your thoughts or experiences if you've worked with something similar.

Thank you in advance!

**Comment by riebl** [comment]
For future reference: #346 :-)
